mavros/src/plugins/manual_control.cpp

```diff
@@ -34,6 +34,7 @@ class ManualControlPlugin : public plugin::PluginBase {
 		PluginBase::initialize(uas_);
 
 		control_pub = manual_control_nh.advertise<mavros_msgs::ManualControl>("control", 10);
+		control_sub = manual_control_nh.subscribe("override", 1, &ManualControlPlugin::override_cb, this);
```

Comment:
Should be called `override_sub` in case topic name `override`. Maybe name it `send`?

Reply:
Yes, `send` is better. But, in such case, there'll be two topics, `send` and `control`, it's non-obvious, that these are send and receive, actually.